tests/test-class-module.php

```diff
@@ -1,10 +1,13 @@
 <?php
 
 class WP_Test_Edit_Flow_Class_Module extends WP_UnitTestCase {
+
 	protected static $admin_user_id;
+	protected static $EditFlowModule;
 
 	public static function wpSetUpBeforeClass( $factory ) {
 		self::$admin_user_id = $factory->user->create( array( 'role' => 'administrator' ) );
+		self::$EditFlowModule = new EF_Module();
```

Comment:
Not strictly necessary, I guess, but it's a good idea to clean up after ourselves (we'd end up with reference after the last test is run).